lonestar/analytics/cpu/betweennesscentrality/BetweennessCentrality.cpp

```diff
@@ -38,35 +38,17 @@ namespace cll = llvm::cl;
 static cll::opt<std::string>
     inputFile(cll::Positional, cll::desc("<input file>"), cll::Required);
 
-static cll::opt<std::string> sourcesToUse("sourcesToUse",
-                                          cll::desc("Whitespace separated list "
-                                                    "of sources in a file to "
-                                                    "use in BC"),
-                                          cll::init(""));
-
-static cll::opt<unsigned int>
-    numOfSources("numOfSources",
-                 cll::desc("Number of sources to compute BC on (default all)"),
-                 cll::init(0));
-
-static llvm::cl::opt<unsigned int>
-    iterLimit("numOfOutSources",
-              llvm::cl::desc("Number of sources WITH EDGES "
-                             " to compute BC on (default is all); does "
-                             "not work with Level BC"),
-              llvm::cl::init(0));
-
-static cll::opt<bool>
-    singleSourceBC("singleSource",
-                   cll::desc("Level: Use for single source BC (default off)"),
-                   cll::init(false));
-
-static cll::opt<uint64_t>
-    startSource("startNode",
-                cll::desc("Level/Outer: Starting source node used for "
-                          "betweeness-centrality (default 0); works with "
-                          "singleSource flag only"),
-                cll::init(0));
+static cll::opt<std::string> startNodesFile(
```

Comment:
Correct me if am wrong but it seems it is reading sources from a file and then using all of them to run BC. For Intel experiments, we need another parameter that specifies how many sources to run bc application each time. So if that number is 4 and total sources are 64 read from a file, I would do 4 at a time and run bc 16 times.